### DISCUSSION TITLE
Running on Debian with transmission

### DISCUSSION THREAD

**Discussion by iCarl** [jesec/flood#292] [Q&A]
I'm trying to get this to work with cli running:

`flood --truser *transmissionuser* --trpass *transmissionpassword* --trurl http://127.0.0.1:9091 --baseuri transmission/rpc --host 10.0.1.5 -p 3000` but when I try to access http://10.0.1.5:3000 all it says is: "Cannot GET /"

I've tried disabling auth, but then I get: " Issue #0: invalid_union at configUser
  Invalid input"

What am I doing wrong?

**Comment by jesec** [comment]
That `--baseuri` is obviously wrong, unless you want to access Flood at `http://10.0.1.5:3000/transmission/rpc`. And, note that client connection settings arguments are only effective if you disable authentication.
